build.fsx

```diff
@@ -73,35 +66,20 @@ let vsceTool =
 
 Target "Clean" (fun _ ->
     CleanDir "./temp"
-    CopyFiles "release" ["README.md"; "LICENSE.md"; "RELEASE_NOTES.md"]
```

Comment:
They were just copies of the files in the root dir, but now the `release` dir is the root dir itself so the copies shouldn't be needed anymore. Of course, if @Krzysztof-Cieslak prefers to still have a `release` dir we can bring them back.